docs/contributors/releasing.md

```diff
@@ -112,18 +119,24 @@ title: Making a release
   - Once the VS Code extension has been updated on the Marketplace, click
     "Publish release".
 
-- Update version numbers in:
-  - `build.sbt` - update `localSnapshotVersion`
-  - `.github/ISSUE_TEMPLATE/bug_report.yml` - update `Version of Metals`
-  - `.github/workflows/mtags-auto-release.yml` - update `metals_version` and `metals_ref`
-
 - Announce the new release with the link to the release notes:
   - on [Discord](https://discord.com/invite/RFpSVth)
 
+## tl;dr release steps
```

Comment:
Do we need tldr? I think it's important to go through all the step exactly as they are.

Reply:
It's more like the sanity check, I can rename it if you don't like the `tl;dr`

Reply:
Looks better, thanks!